plugins/reporters/web-app-template/src/App.css

```diff
@@ -249,4 +264,4 @@ tr.ant-table-expanded-row, tr.ant-table-expanded-row:hover {
     background-repeat: no-repeat;
-}
+}
```

Comment:
Please re-introduce this final newline.

Reply:
what do you mean by "re-introduce" it ?

Reply:
See the diff:

![image](https://user-images.githubusercontent.com/349154/234865823-14912a30-3abd-4838-9830-fcab392498d9.png)

It was there before, but you've removed it without reason, so you should re-introduce it.

Reply:
I just added new lines to the file so the final line moved to line 257

Reply:
Having new lines in the file is fine, that's not what I meant. The problem is this red icon on the right-hand side in the green area of the diff. It indicates that you removed a final newline character from the end of the file. That single character is what you need to restore.

Reply:
Done